projects/sunbird-video-player/src/lib/services/viewer.service.ts

```diff
@@ -80,7 +82,26 @@ export class ViewerService {
       this.mimeType = metadata.mimeType;
     }
   }
-
+  handleTranscriptsData(transcripts) {
+    if (_.isArray(transcripts)) {
```

Comment:
if (!_.isArray(transcripts)) { 
this.raiseExceptionLog('INVALID_TRANSCRIPT_DATATYPE', 'TRANSCRIPT', new Error('Transcript data should be array'), this.traceId);
return [];
}